src/app/services/redux-constants.ts

```diff
@@ -4,3 +4,5 @@ export const CLEAR_QUERY_ERROR = 'CLEAR_QUERY_ERROR';
 export const GET_AUTH_TOKEN_SUCCESS = 'GET_AUTH_TOKEN_SUCCESS';
 export const LOGOUT_SUCCESS = 'LOGOUT_SUCCESS';
 export const QUERY_GRAPH_RUNNING = 'QUERY_GRAPH_RUNNING';
+export const HEADER_ADD = 'HEADER_ADD';
```

Comment:
There are two groups of constants, SUCCESS and ERROR

Rename these to either HEADER_ADD_SUCCESS/ERROR for consistency.